pepdbagent/db_utils.py

```diff
@@ -119,6 +121,10 @@ class Projects(Base):
 
     namespace_mapping: Mapped["User"] = relationship("User", back_populates="projects_mapping")
 
+    history_mapping: Mapped[List["HistoryProjects"]] = relationship(
+        back_populates="project_mapping", cascade="all, delete-orphan"
+    )  # TODO: check if cascade is correct
```

Comment:
Does this need implementing in this PR? I see its a TODO

Reply:
done